identity/sharedid.md

```diff
@@ -33,9 +43,12 @@ the module:
 pbjs.setConfig({
     userSync: {
         userIds: [{
-            name: 'sharedId',
+            name: 'sharedId', //"pubCommonId" as a name is supported for backwards compatibility,
+            params: {
+                pixelUrl: "/wp-json/pubcid/v1/extend/"
```

Comment:
I would add: '// this parameter identifies your server side endpoint that will set a first party cookie if you have configued SharedId server'

Reply:
done